src/index.js

```diff
@@ -0,0 +1,156 @@
+'use strict'
```

Comment:
Looks like there are several elements scattered around the file, along with some repetitive code for registering event listeners individually.

Let's streamline this by creating one function that will contain any elements that need an event listener:

```js
const registerEventHandler = () => {
    const skyOptions = document.getElementById('skySelect');
    skyOptions.addEventListener('change', changeSky);

    const decreaseTempButton = document.getElementById('decreaseTempControl');
    decreaseTempButton.addEventListener('click', decreaseTemp);
    // or: decreaseTempHandler(); since you have a function already built for this
    // so on and so forth
};

// then, likely placed at the bottom of the file
document.addEventListener('DOMContentLoaded', registerEventHandler);
```

Reply:
The rest of the variables that grab the values we wish to calculate or re-render, let's collect them together for the sake of organization, or collect them all into one object of key-value pairs.

Just like the event and event listeners we registered in `registerEventHandler`, we can create an object of initial state:
```js
const state = {
    tempValue: 0,
    currentCity: 'Seattle',
    landscape: null
};

// then inside the registerEventHandler
state.tempValue = parseInt(document.getElementById('tempValue').textContent);
state.landscape = document.getElementById('landscape');
// so on and so forth
```

This isn't necessary, but it would create some structure and organization for other coders when reading your code.